release.js

```diff
@@ -23,14 +23,15 @@ const semver = require('semver');
 const replace = require('replace-in-file');
 const releaseit = require('release-it');
 
-const pkg = require('./package.json');
+const versions = require('./versions.json');
```

Comment:
decided to create a new file to keep the versions, rather than relying on the package.json. this will be helpful when we move away from npm and allows us to keep multiple versions